core/src/main/java/org/infinispan/interceptors/impl/TransactionalExceptionEvictionInterceptor.java

```diff
@@ -90,7 +91,9 @@ public void inject(Configuration config, CacheNotifier cacheNotifier,
    @Start
    public void start() {
       if (memoryConfiguration.storageType() == StorageType.OFF_HEAP) {
-         minSize = UnpooledOffHeapMemoryAllocator.estimateSizeOverhead(memoryConfiguration.addressCount() << 3);
+         // TODO: this is technically not correct - as the underlying map can resize (also it doesn't take int account
+         // we have a different map for each segment when not in LOCAL mode
```

Comment:
Yeah, things got a little complicated since you first wrote this :)

Reply:
I created https://issues.jboss.org/browse/ISPN-10637 to handle this btw.